index.js

```diff
@@ -4,15 +4,49 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+updateTitle = header.style.color = "Green"
```

Comment:
Although this works, there's no need to assign `header.style.color = "Green"` to `updateTitle`.  You can just have: 

```javascript
header.style.color = "Green"
```